python/arcticdb/version_store/library.py

```diff
@@ -810,6 +810,38 @@ def update(
             prune_previous_version=prune_previous_versions,
         )
 
+    def list_symbols_with_staged_data(self) -> List[str]:
+        """
+        Removes staged data.
```

Comment:
Comment is incorrect

Reply:
Function removed as James noticed this functionality already exists:
https://github.com/man-group/ArcticDB/pull/778#discussion_r1300340199